actions/packages.py

```diff
@@ -233,9 +235,11 @@ def __transact_args__(self, transaction_data):
                 assert False, "Unknown package transaction action."
         return args
 
-    def patch_install(self, patch_list):
+    def patch_install(self, patch_list, allow_vendor_change=False):
         args = ["-n", "-x", "install", "--auto-agree-with-licenses"]
 
+        if allow_vendor_change:
```

Comment:
Same consideration as above about checking the Zypper version before injecting the vendorchange flag.